src/Core/Math/Ellipsoid.js

```diff
@@ -1,182 +1,171 @@
-/**
- * Generated On: 2015-10-5
- * Class: Ellipsoid
- * Description: Classe mathématique de  l'ellispoide
- */
-
-
 import * as THREE from 'three';
+import proj4 from 'proj4';
 import Coordinates from 'Core/Geographic/Coordinates';
 
-function Ellipsoid(size) {
-    // Constructor
-
+export const ellipsoidSizes = {
+    x: proj4.WGS84.a,
+    y: proj4.WGS84.a,
+    z: proj4.WGS84.b,
+};
 
-    this.rayon_1 = size.x;
-    this.rayon_2 = size.y;
-    this.rayon_3 = size.z;
+const normal = new THREE.Vector3();
 
-    this.size = new THREE.Vector3(size.x, size.y, size.z);
+class Ellipsoid {
+    constructor(size = ellipsoidSizes) {
+        this.rayon_1 = size.x;
```

Comment:
could add test if all values doesn't equal to zero.

Reply:
I don't see the test to add, sorry.